app/controllers/projects_controller.rb

```diff
@@ -343,6 +343,6 @@ def return_context
     render_404 && return if @project.blank?
     return unless @project.deleted?
     flash[:alert] = 'The project you requested had been deleted.'
-    redirect_to user_path(@user)
+    check_redirect "/#{@user.username}"
```

Comment:
>  This part is all ready to be replaced... I have already defined the routes of the projects. Issues not yet merged but they're also ready.

can please confirm by following your branch on hakiri that changes you made solves the redirect warning?